src/components/Attachments/AttachmentView/index.tsx

```diff
@@ -100,33 +87,35 @@ function AttachmentView({
     transaction,
     reportActionID,
     isHovered,
-    optionalVideoDuration,
-}) {
+    duration,
+}: AttachmentViewProps) {
+    const {translate} = useLocalize();
     const {updateCurrentlyPlayingURL} = usePlaybackContext();
     const theme = useTheme();
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const [loadComplete, setLoadComplete] = useState(false);
-    const isVideo = (typeof source === 'string' && Str.isVideo(source)) || (file && Str.isVideo(file.name));
+    const isVideo = (typeof source === 'string' && Str.isVideo(source)) || (file?.name && Str.isVideo(file.name));
 
     useEffect(() => {
         if (!isFocused && !(file && isUsedInAttachmentModal)) {
             return;
         }
-        updateCurrentlyPlayingURL(isVideo ? source : null);
-    }, [isFocused, isVideo, source, updateCurrentlyPlayingURL, file, isUsedInAttachmentModal]);
+        updateCurrentlyPlayingURL(isVideo && typeof source === 'string' ? source : null);
+    }, [file, isFocused, isUsedInAttachmentModal, isVideo, source, updateCurrentlyPlayingURL]);
 
     const [imageError, setImageError] = useState(false);
 
     useNetwork({onReconnect: () => setImageError(false)});
 
     // Handles case where source is a component (ex: SVG) or a number
     // Number may represent a SVG or an image
-    if ((maybeIcon && typeof source === 'number') || _.isFunction(source)) {
-        let iconFillColor = '';
-        let additionalStyles = [];
-        if (isWorkspaceAvatar) {
-            const defaultWorkspaceAvatarColor = StyleUtils.getDefaultWorkspaceAvatarColor(file.name);
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Huh, this applies to if statements?

Reply:
Very strange, changed the order of values in if and it no longer throws eslint errror 😃

Reply:
Great!